omod/src/main/webapp/resources/htmlforms/073-ChildRegister.xml

```diff
@@ -0,0 +1,497 @@
+<htmlform 
+formName="HMIS 073 Child Form" 
+formDescription="Captures children information" 
+formEncounterType="2d7b9d91-2181-4c6d-8b17-3de9cd33f402" 
+formUuid="678721ff-c0f4-488c-bfb6-5cf01a2c2136" 
+formVersion="1.0" 
+formAddMetadata="yes" 
+formUILocation="patientDashboard.visitActions" 
+formOrder="7" 
+formIcon="icon-medkit" 
+formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.age &lt; 15 &amp;&amp;
```

Comment:
Add &amp;&amp; sessionLocation.uuid=='e9bc61b5-69ff-414b-9cf0-0c22d6dfca2b' so that the form only shows in MCH clinic